.github/workflows/all-post-commit-workflows.yaml

```diff
@@ -24,6 +24,21 @@ jobs:
     with:
       tracy: true
     secrets: inherit
+  # UMD Unit Tests
+  umd-unit-tests:
+    secrets: inherit
+    strategy:
+      fail-fast: false
+      matrix:
+        test-group: [
+          { arch: grayskull, runner-label: E150 },
```

Comment:
would add "in-service" label so we don't grab any disabled runners too

Reply:
I have the `in-service` label further down in `umd-unit-tests.yaml` since that seems like the setup used for other pipelines?